solidity/dashboard/src/reducers/liquidity-rewards.js

```diff
@@ -0,0 +1,57 @@
+const wrappedTokenInitialData = {
+  shareOfPoolInPercent: 0,
+  reward: 0,
+  wrappedTokenBalance: 0,
+  lpBalance: 0,
+  isFetching: false,
+  error: null,
+}
+
+const initialState = {
+  KEEP_ETH: { ...wrappedTokenInitialData },
+  TBTC_ETH: { ...wrappedTokenInitialData },
+  KEEP_TBTC: { ...wrappedTokenInitialData },
+}
+
+const liquidityRewardsReducer = (state = initialState, action) => {
+  if (!action.payload) {
+    return state
+  }
+
+  const { liquidityRewardPair, ...restPayload } = action.payload
```

Comment:
Hmmm... I would personally change the `liquidityRewardPair` variable name to `liquidityRewardPairName`. The `liquidityRewardPair` name may be misinterpreted as `LIQUIDITY_REWARD_PAIR` object from constants.js. 👍 if you agree or 👎 if you disagree.

Reply:
b09f05e